packages/components/src/modal/index.js

```diff
@@ -101,6 +102,11 @@ class Modal extends Component {
 		ariaHelper.showApp();
 	}
 
+	onModalOnly( ev ) {
```

Comment:
The name is not entirely obvious as to the purpose it serves. I might expect something even as simple as `stopPropagation`, or `stopEventPropagationOutsideModal` for more explicitness / clarity.